evil-colemak-basics.el

```diff
@@ -118,6 +128,14 @@ rotated; see evil-colemak-basics-rotate-t-f-j."
           "j" 'evil-snipe-j
           "J" 'evil-snipe-J))
        (t (user-error "Invalid evil-colemak-basics-char-jump-commands configuration"))))
+    (when (eq evil-colemak-basics-layout-mod 'mod-dh)
+      (evil-define-key '(motion normal visual) keymap
+	"m" 'evil-next-line)
```

Comment:
i think you would need more here? the normal keymap has these in place:

```
      "n" 'evil-next-line
      "gn" 'evil-next-visual-line
      "gN" 'evil-next-visual-line
```

Reply:
Oops, this is a typo that I added during a refactor; it should be 'evil-backward-char (since it's mnei instead of hnei/hjkl).